tutorials/alcf/02_ThetaGPU_ray/README.rst

```diff
@@ -124,21 +116,13 @@ Edit the ``#COBALT ...`` directives:
     #COBALT -n 2
     #COBALT -q full-node
     #COBALT -t 20
+    #COBALT --attrs filesystems=home,grand,eagle,theta-fs0
 
 and adapt the executed Python application depending on your needs:
 
-.. code-block:: bash
+.. code-block:: python
 
-    ssh -tt $head_node_ip "source $INIT_SCRIPT && cd $EXP_DIR && \
-        deephyper nas random \
-        --problem deephyper.benchmark.nas.linearRegHybrid.Problem \
-        --evaluator ray \
-        --run-function deephyper.nas.run.run_base_trainer \
-        --num-workers -1 \
-        --ray-address auto \
-        --ray-num-cpus-per-task 1 \
-        --ray-num-gpus-per-task 1 \
-        --verbose 1"
+    myscript.py
 
 Finally, submit the script from a ThetaGPU login node (e.g., ``thetagpusn1``):
```

Comment:
outdated, now we have to use `qsub-gpu` also mentionne that for the installation we need to request a `single-gpu` node (or add this in installation for ThetaGPU)

Reply:
I wasn't aware of this ```qsub-gpu`` command